Apps/W1/External File Storage - Azure File Service Connector/app/src/ExtFileShareAccountWizard.Page.al

```diff
@@ -0,0 +1,155 @@
+    SourceTable = "Ext. File Share Account";
+    SourceTableTemporary = true;
+    Permissions = tabledata "Ext. File Share Account" = rimd;
+    PageType = NavigatePage;
+    Extensible = false;
+    Editable = true;
+    ApplicationArea = All;
+
+    layout
+    {
+        area(Content)
+        {
+            group(TopBanner)
+            {
+                Editable = false;
+                ShowCaption = false;
+                Visible = TopBannerVisible;
+                field(NotDoneIcon; MediaResources."Media Reference")
+                {
+                    Editable = false;
+                    ShowCaption = false;
+                    ToolTip = ' ';
+                    Caption = ' ';
+                }
+            }
+
+            field(NameField; Rec.Name)
+            {
+                Caption = 'Account Name';
+                ToolTip = 'Specifies the name of the Azure File Share account.';
+                ShowMandatory = true;
+                NotBlank = true;
+
+                trigger OnValidate()
+                begin
+                    IsNextEnabled := FileShareConnectorImpl.IsAccountValid(Rec);
+                end;
+            }
+
+            field(StorageAccountNameField; Rec."Storage Account Name")
+            {
+                Caption = 'Storage Account Name';
+                ShowMandatory = true;
+
+                trigger OnValidate()
+                begin
+                    IsNextEnabled := FileShareConnectorImpl.IsAccountValid(Rec);
+                end;
+            }
+
+            field("Authorization Type"; Rec."Authorization Type")
+            {
+            }
+
+            field(SecretField; Secret)
+            {
+                Caption = 'Secret';
+                ExtendedDatatype = Masked;
+                ToolTip = 'Specifies the Shared access signature Token or SharedKey.';
+                ShowMandatory = true;
+            }
+
+            field(FileShareNameField; Rec."File Share Name")
+            {
+                Caption = 'File Share Name';
+                ToolTip = 'Specifies the file share to use of the storage account.';
+                ShowMandatory = true;
+
+                trigger OnValidate()
+                begin
+                    IsNextEnabled := FileShareConnectorImpl.IsAccountValid(Rec);
+                end;
+            }
+        }
+    }
+
+    actions
+    {
+        area(processing)
+        {
+            action(Back)
+            {
+                Caption = 'Back';
+                ToolTip = 'Back';
+                Image = Cancel;
+                InFooterBar = true;
+
+                trigger OnAction()
+                begin
+                    CurrPage.Close();
+                end;
+            }
+
+            action(Next)
+            {
+                Caption = 'Next';
+                Image = NextRecord;
+                Enabled = IsNextEnabled;
+                InFooterBar = true;
+                ToolTip = 'Next';
```

Comment:
The ToolTip for Next and Back could be improved.
e.g. :
- `ToolTip = 'Move to the previous step.';`
- ToolTip = 'Move to the next step.';
 
I know that other wizards are there bad, too.

Reply:
I reviewed this again. 
Isn't the Naming here wrong?
The first action cancels the wizard and the 2nd Finish the wizards..?